CONTRIBUTING.md

```diff
@@ -107,3 +107,38 @@ Icons are located in [packages/react/src/components/Icon/icons](./packages/react
 - Remove any namespace attributes (such as `xmlns:*`) from the SVG.
 - Save your icon in the above location with the proposed name of your icon, e.g. `[your-name].svg`. The `your-name` portion will be how the icon is used in the React Component, e.g. `<Icon type="your-name" />`.
 - Submit a new `feat` PR with your changes.
+
+## Breaking Changes
+
+Cauldron is constantly evolving, and it's inevitable that breaking changes will need to be introduced. Before a breaking change is released, other alternatives should be considered and allow for a deprecation period before the breaking change is released.
+
+### Components
+
+For cauldron's component library, any of the following would be considered breaking changes:
+
+* Renaming (exported name) or removing a Component
+* Changing a prop that would break usage of an existing component
+* Adding a new required prop to an existing component
+
+### Styles
+
+For cauldron's styles library, any of the following would be considered breaking changes:
+
+* Changing or removing a class name
+* Changing or removing a global variable (see `variables.css`)
+* Changing or removing a component variables
+
+In addition, changing a css property may be a breaking change for the following:
+
+* Changing a global variable property (see `variables.css`)
+* Changing a component variable property
```

Comment:
This is a little unclear to me what you are referring to.

> In addition, changing a css property may be a breaking change for the following:

Sounds like a CSS property value is changed maybe? (e.g. `color: blue; -> color: red;`)

> Changing a global variable property

This sounds like changing the value of a CSS variable (e.g. `--background-color: blue; -> --background-color: red;`. And if that's the case, how is that different than what is called out above "Changing or removing a global variable". Or is what is called out above more about "changing the name of" instead of "changing the value of"?

Reply:
It's the property portion of a css variable. Changing the name will always be a breaking change because someone consuming cauldron could be using that variable name in their code, so changing a name from `--foo` to `--bar` would mean that css property using that variable would become unset.

Changing the property value of a css variable isn't as clear cut which is why the additive section is slightly different and requires a more in-depth review from a member of the cauldron team. A good example of this is that we recently changed the global `--focus` property value to better contrast against all of our standard background colors. This was not a breaking change as we already identified that we has some low contrast accessibility issues within cauldron. But that's not to say there aren't other property values that would be a breaking change, thus requiring a more in-depth review before allowing.

Reply:
If I understood correctly, then IMO, this would be more clearly written as:

```suggestion
In addition, changing a css variable value may be a breaking change for the following:

* Changing a global variable value (see `variables.css`)
* Changing a component variable value
```

Since a variable IS a custom property, but it's the value of the variable/custom property that is being referred to here.

Reply:
Official documentation calls them _css custom properties_, but I can be more specific here.

Reply:
I'm just trying to distinguish between the _name_ of the custom property/variable (e.g. `--focus`) and the _value_ of that variable (e.g. `blue`)

Reply:
Yes, I updated the words here 🙃